backend/framer.go

```diff
@@ -0,0 +1,22 @@
+package backend
+
+import "github.com/grafana/grafana-plugin-sdk-go/data"
+
+// FrameResponse creates a backend.DataResponse that contains the Framer's data.Frames
```

Comment:
Isn't it less confusing to drop the `backend.` prefix, since DataResponse is in the same package?
```suggestion
// FrameResponse creates a DataResponse that contains the Framer's data.Frames
```